config/fortify-client.properties

```diff
@@ -0,0 +1 @@
+fortify.client.releaseId=65430
```

Comment:
You need to use the release id of send-letter-service here. ID should be 65508, but better to double check.

Reply:
I have updated the SLS release id